spec/spec_helper.rb

```diff
@@ -43,6 +43,8 @@
   config.order = :random
 end
 
+RSpec::Matchers.define_negated_matcher :not_change, :change
+
 Shoulda::Matchers.configure do |config|
```

Comment:
I'm guessing `not_to change` didn't work?  https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/change-matcher#expect-no-change

Reply:
Can't chain not_to change